Userland/Games/Hearts/Game.cpp

```diff
@@ -161,6 +161,26 @@ void Game::timer_event(Core::TimerEvent&)
     update();
 }
 
+bool Game::other_player_has_lower_value_card(Player& player, Card& card)
+{
+    for (auto& other_player : m_players)
+        if (&player != &other_player)
+            for (auto& other_card : other_player.hand)
+                if (other_card && card.type() == other_card->type() && hearts_card_value(*other_card) < hearts_card_value(card))
+                    return true;
```

Comment:
Please use `{}` for any if/for that as a body of more than one line.

Reply:
Fixed. :)